tests/ProxyManagerTest/Functional/LazyLoadingValueHolderFunctionalTest.php

```diff
@@ -402,7 +406,11 @@ private function createInitializer(string $className, object $realInstance, ?Moc
                 );
         }
 
-        return static function (
+        return
+        /**
+         * @psalm-param ?TClass $wrappedObject
```

Comment:
Similar to above, closure parameter should accept any object here.

The factory states:

```
     * @psalm-param Closure(
     *   RealObjectType|null=,
     *   RealObjectType&ValueHolderInterface<RealObjectType>&VirtualProxyInterface=,
     *   string=,
     *   array<string, mixed>=,
     *   ?Closure=
     * ) : bool $initializer
```

In theory, `function (?object $object) : bool {...` should be compatible, no?